packages/account/src/Sections/Security/DeactivateAccount/__test__/deactivate-account.spec.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import { MemoryRouter } from 'react-router-dom';
```

Comment:
any reason that we use MemoryRouter instead of BrowserRouter?

Reply:
MemoryRouter has no Browser Dependencies. It will speed up the test execution compare to BrowserRouter.